### PR TITLE
Completed GET /api/articles and its tests

### PR DESCRIPTION
**Pull request by Kingsl8yer**
GET /api/articles completed and its tests!

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -93,3 +93,34 @@ describe("GET /api/articles/:article_id", () => {
...
+              expect(typeof article.article_id).toBe("number");
+              expect(typeof article.topic).toBe("string");
+              expect(typeof article.created_at).toBe("string");
+              expect(typeof article.votes).toBe("number");
+              expect(typeof article.article_img_url).toBe("string");
+              expect(typeof article.comment_count).toBe("string");
+          });
+      });
+  });
+
```
Comment (Augs0) [inline __tests__/index.test.js:114]: Nice testing so far. We need to also test the default sort and order are as expected (created_at, desc). We can use jest-sorted for this

File: __tests__/index.test.js [hunk 2]
```diff
@@ -93,3 +93,34 @@ describe("GET /api/articles/:article_id", () => {
...
+  //Error handling
+  test("status 404: responds with error message when given a wrong URL", () => {
+      return request(app)
```
Comment (Augs0) [inline __tests__/index.test.js]: We only need one test for the 404 as it's for all our endpoints. You already this before so this test can be removed.

**Review by Augs0 — CHANGES_REQUESTED** [review]
Nice work Miguel. Can you add one more test here making sure the default sort and order are correct?

**Review by B7Bprog — APPROVED** [review]
Hey Miguel, Nice new test, please merge! 🎉

**Pull request merged**